queryscript/src/compile/sql.rs

```diff
@@ -2408,8 +2411,22 @@ pub fn compile_sqlexpr(
                     }
 
                     let func_expr = func.expr.unwrap_schema_entry().await?;
+                    let compiled_func_expr = match func_expr {
+                        Expr::UncompiledFn(def) => {
+                            let (compiled_body, _) = compile_fn_body(
```

Comment:
should we assert something about the generics here? I assume they should be empty?